content/pages/legal.md

```diff
@@ -1,172 +1,161 @@
+- In some counties, you may need to schedule a hearing before the probate judge to complete the name change.
+- Check your local probate court office’s website or give them a phone call for further details.
 
-Once the name change is complete, you’ll end up with certified copies of
-the completed name change Court Order. They’ll have an embossed seal on 
-them from the probate court. 
+Once the name change is complete, you’ll end up with certified copies of the completed name change court order.
 
-Hang onto these! You will need them later.
+- They’ll have a seal on them from the probate court.
+- **Keep these someplace safe! You will need them.**
 
-## After you get your court order
+## After the court order
 
-Once you have a court order for your name change, [the first place to visit is
-Social Security][7]. 
+Once you have a court order for your name change,
+[the first place to visit is Social Security][form-ssa-name].
 
-If you're able to print, you can save time by completing page 5 of [Form SS-5][2]
-in advance. At this stage, it is possible to simultaneously [update your gender marker][1] 
-with them when submitting Form SS-5 (see below Gender Marker section for details).
-This must be done eventually if changing your gender marker, 
-and can be completed without providing any medical documentation. Checking 
-this box now may save you time later.
+When you update your name with Social Security, they will give you a
+letter indicating that you requested changes.
 
-When you update with Social Security they will give you a letter indicating that 
-you requested changes. This letter is important, so hold on to it!
+- This letter is important, so be sure to keep it someplace safe.
 
-24 hours after updating with Social Security you can update your driver's
-license. Most DMVs should be able to change your name. Be sure to bring your
-letter from Social Security in addition to your court order. While you're at
-the DMV, you should also update your car registration and title.
+24 hours after updating with Social Security, you can update your driver's
+license.
 
-Once you've updated your driver's license you should start updating all the
-other places that know your name. Some important places to notify:
+- Most DMVs should be able to change your name.
+- Be sure to bring your letter from Social Security in addition to your original court order.
+- You should also update your voter registration
+  information and car registration/title if applicable.
 
- - Your employer's HR department
- - Banks
- - Credit card companies
- - And many more (it's amazing how many people know your name!)
+Once you've got a new ID, you can start elsewhere. Some important places:
+
+- Your employer's HR department
+- Bank and credit card companies
+- Your car insurance company
+- Your health insurance company
+- Your school or university
+- Doctors and/or medical providers
+- Your local post office (setup mail forwarding!)
```

Comment:
I'm not sure I follow this one. You don't have to go to the post office do you? I didn't. I just get mail in my old and new names.

Reply:
I actually had to at first because my mail carrier was very picky at what they did/didn't deliver, but this was years ago and he had been easily working that job for 20+ years, so it's probably not like that anymore